release notes/v0.37.0.md

```diff
@@ -0,0 +1,71 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring.
+
+## New Features!
+
+### Added an option to output k6 logs to a file through --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+This is on top of the already supported options for sending logs to stdout/stderr and to [Grafana Loki](https://grafana.com/oss/loki/). This new feature speaks for itself with simple usage examples:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+And one more with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions ([an example](https://github.com/MStoykov/xk6-events)). In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first, it's likely that it will be used in some experimental extensions.
+
+If you are an extension developer, please use it and give your feedback. But take into consideration that it's likely that the current Go API may change.
```

Comment:
for giving more emphasis

```suggestion
> :warning: If you are an extension developer, please use it and give your feedback. But take into consideration that it's likely that the current Go API may change.
```

or maybe could it be a text code block? :thinking: